src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -184,7 +184,7 @@ class NotebookCard extends Component {
 }
 
 class WorkspaceNotebooks extends Component {
-  async componentWillMount() {
+  async refresh() {
```

Comment:
Need to reset `clusters`/`cluster` here or things will bomb

Reply:
`loadClusters()` does that